lib/plugins/acts_as_commentable/comment_methods.rb

```diff
@@ -54,7 +54,7 @@ def children_count
     # Returns comment itself if unthreaded
     def full_set
       if self.threaded_left
-        Comment.include(:pseud).where("threaded_left BETWEEN (?) and (?) AND thread = (?)",
+        Comment.includes(:pseud).where("threaded_left BETWEEN (?) and (?) AND thread = (?)",
```

Comment:
This doesn't technically need to be in here since I stopped using `full_set`, but it seemed better to keep these fixes than not.